adopter_bot/settings.py

```diff
@@ -0,0 +1,218 @@
+SECRET_KEY = os.getenv("SECRET_KEY")
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ['localhost', '127.0.0.1', 'c5ad-95-251-43-142.ngrok-free.app']
+
+
+# Application definition
+
+DJANGO_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    "django_celery_beat",
+    'django_filters',
+    'rest_framework',
+    'drf_spectacular',
+    'rest_framework.authtoken',
+    'rest_framework_simplejwt',
+
+]
+
+
+PROJECT_APPS = [
+    'adopter.apps.AdopterConfig',
+    'pet.apps.PetConfig',
+    'adoption_request.apps.AdoptionRequestConfig',
+    'temporal_owner.apps.TemporalOwnerConfig',
+    ]
+
+THIRD_PARTY_APPS = [
+    "debug_toolbar",
+    ]
+
+INSTALLED_APPS = DJANGO_APPS + PROJECT_APPS + THIRD_PARTY_APPS
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
+]
+
+ROOT_URLCONF = 'adopter_bot.urls'
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [BASE_DIR/"templates"],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    },
+]
+
+WSGI_APPLICATION = 'adopter_bot.wsgi.application'
+
+REST_FRAMEWORK = {
+    'PAGE_SIZE': 10,
+    'DEFAULT_PAGINATION_CLASS': "rest_framework.pagination.PageNumberPagination",
+    'DEFAULT_FILTER_BACKENDS': ["django_filters.rest_framework.DjangoFilterBackend"],
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
+    ],
+    'DEFAULT_SCHEMA_CLASS': 'drf_spectacular.openapi.AutoSchema',
+}
+
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(minutes=123321),
+    "REFRESH_TOKEN_LIFETIME": timedelta(days=1),
+    "ROTATE_REFRESH_TOKENS": True,
+}
+
+AUTH_USER_MODEL = 'temporal_owner.TemporalOwner'
+AUTHENTICATION_BACKENDS = [
+    'django.contrib.auth.backends.ModelBackend',
+    'temporal_owner.backends.TemporalOwnerBackend',
+]
+# Database
+# https://docs.djangoproject.com/en/4.2/ref/settings/#databases
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "HOST": os.environ["POSTGRES_HOST"],
```

Comment:
додайте змінну до .env.template

Reply:
Навіщо,це ж стандартні налаштування,які залежать від середовища, на якому будуть запускати цей додаток?

Reply:
Не розумію , в файлі env.templates є усе окрім "HOST", тому що: The POSTGRES_HOST value should be specific to the environment where the application is deployed. In the .env.template, it is not necessary to provide a specific value for POSTGRES_HOST as it will depend on the deployment environment.

Reply:
просто додайте туди цю змінну, без вказання значення. Це необхідно для того, щоб запускаючи проект людина могла зрозуміти що ця змінна необхідна. Значення цієї змінної буде встановлено відповідно до системи, на якій проект буде запускатися. Але наявність змінної в .env.template повідомить, що ця змінна має бути і її потрібно буде встановити:
```
POSTGRES_HOST=your_host
```

Reply:
А,ок,я думала,ви мали на увазі,щоб я конкретну адресу вказала